src/UXClient/Components/Hierarchy/Hierarchy.ts

```diff
@@ -132,8 +132,8 @@ class Hierarchy extends Component {
                         this.contextMenu.append('div').text(`${noTildeName}`).on('click', () => n.click(n));
                     })
                     this.contextMenu.classed('tsi-hierarchyContextMenu', true);
-                    let mouseWrapper = d3.mouse(this.hierarchyList.node());
-                    let mouseElt = d3.mouse(el.node());
+                    let mouseWrapper = d3.pointer(event, this.hierarchyList.node());
+                    let mouseElt = d3.pointer(event, el.node());
```

Comment:
could there be an issue with how d3 handles this if `event` is `undefined`? looks like `event` param can be optional to this function

Reply:
Yeah potentially it result in a null pointer exception in d3. Looks like the only thing it does is to add some styling, so I've added a null check for `event`.